### PR TITLE
Some map tweaks

### PR DESCRIPTION
**Pull request by Kurgis**
## About The Pull Request

Some pipes in the Engine Room have been renamed for clarity, and the Wasting Room vent has been set-up properly from the get-go.

The disposal "Yeetlet" should no longer get mobs stuck when the shields are on. Certain spots near the IH Lobby where walls were showing under open space, were covered with hulling.

A EFTPOS scanner was added to the Bar Backroom.

### CODE REVIEW

File: maps/CEVEris/_CEV_Eris.dmm [hunk 1]
```diff
@@ -108323,7 +108325,7 @@
 "iDJ" = (
 /obj/machinery/atmospherics/binary/passive_gate{
 	dir = 4;
-	name = "Waste to Space"
+	name = "Waste to Wasting"
```
Comment (ACCount12) [inline maps/CEVEris/_CEV_Eris.dmm]: I don't think that's a good name. Same on all other instances.

Comment (Kurgis) [reply maps/CEVEris/_CEV_Eris.dmm]: I'm open to suggestion, I could rename it to "Waste to Port".

Comment (Kurgis) [reply maps/CEVEris/_CEV_Eris.dmm]: This was after people reporting that Core to Space was "broken", but it doesn't go to Space, it goes to a port which then takes it away, but it starts off, and needs to be set-up.

Comment (TorinoFermic) [reply maps/CEVEris/_CEV_Eris.dmm]: You could use the word Outside and http://www.photographers1.com/Sailing/ShipboardDirections.gif to describe where the pipe leads to.

**Comment by MadmanMartian** [comment]
Thought we were waiting on a better name for the pipes on this one?

**Comment by SpadesSlick** [comment]
I don't think that waiting one month is good thing. It will work for now.

**Pull request merged**